misc/config_definitions.json

```diff
@@ -593,6 +593,10 @@
                 "mpls"
             ]
         },
+        "base_url": {
+            "type": "text",
+            "default": ""
```

Comment:
This one should not actually have a default as the default is set programatically.

Reply:
removed default value